back/boxtribute_server/cli/remove_base_access.py

```diff
@@ -168,6 +181,26 @@ def _update_user_data_in_database(
         (int(base_id),),
     )
 
+    # Soft-delete governing organisation if no active bases left
+    db.database.execute_sql(
+        """\
+UPDATE organisations
```

Comment:
UPDATE organisations o
LEFT JOIN camps c ON c.organisation_id = o.id
SET o.deleted = UTC_TIMESTAMP()
WHERE o.id = (
    SELECT organisation_id 
    FROM camps 
    WHERE id = %s
)
AND (
    SELECT COUNT(*) 
    FROM camps c2 
    WHERE c2.organisation_id = o.id 
    AND (c2.deleted IS NULL OR NOT c2.deleted)
) = 0;